### ISSUE TITLE
Incorrect google font loading through postMessage module

### ISSUE THREAD

**Issue by NazarkinRoman** [themeum/kirki#1988]
### Issue description:
When I use typography control with `transport` set to `auto` each time I select new font it loads with following code from `modules/postmessage/class-kirki-modules-postmessage.php`:

```
/**
 * Processes script generation for typography fields.
 *
 * @access protected
 * @since 3.0.0
 * @param array $args  The arguments for this js_var.
 * @param array $field The field arguments.
 */
protected function script_var_typography( $args, $field ) {

	$args = $this->get_args( $args );

	$script = '';
	$css    = '';

	// Load the font using WenFontloader.
	// This is a bit ugly because wp_add_inline_script doesn't allow adding <script> directly.
	$webfont_loader = 'sc=\'a\';jQuery(\'head\').append(sc.replace(\'a\',\'<\')+\'script>if(!_.isUndefined(WebFont)&&fontFamily){WebFont.load({google:{families:["\'+fontFamily.replace( /\"/g, \'&quot;\' )+\':\'+variant+\'cyrillic,cyrillic-ext,devanagari,greek,greek-ext,khmer,latin,latin-ext,vietnamese,hebrew,arabic,bengali,gujarati,tamil,telugu,thai"]}});}\'+sc.replace(\'a\',\'<\')+\'/script>\');';

	// Add the css.
	$css_build_array  = array(
		'font-family'     => 'fontFamily',
		'font-size'       => 'fontSize',
		'line-height'     => 'lineHeight',
		'letter-spacing'  => 'letterSpacing',
		'word-spacing'    => 'wordSpacing',
		'text-align'      => 'textAlign',
		'text-transform'  => 'textTransform',
		'text-decoration' => 'textDecoration',
		'color'           => 'color',
		'font-weight'     => 'fontWeight',
		'font-style'      => 'fontStyle',
	);
	$choice_condition = ( isset( $args['choice'] ) && '' !== $args['choice'] && isset( $css_build_array[ $args['choice'] ] ) );
	$script          .= ( ! $choice_condition ) ? $webfont_loader : '';
	foreach ( $css_build_array as $property => $var ) {
		if ( $choice_condition && $property !== $args['choice'] ) {
			continue;
		}
		// Fixes https://github.com/aristath/kirki/issues/1436.
		if ( ! isset( $field['default'] ) || (
			( 'font-family' === $property && ! isset( $field['default']['font-family'] ) ) ||
			( 'font-size' === $property && ! isset( $field['default']['font-size'] ) ) ||
			( 'line-height' === $property && ! isset( $field['default']['line-height'] ) ) ||
			( 'letter-spacing' === $property && ! isset( $field['default']['letter-spacing'] ) ) ||
			( 'word-spacing' === $property && ! isset( $field['default']['word-spacing'] ) ) ||
			( 'text-align' === $property && ! isset( $field['default']['text-align'] ) ) ||
			( 'text-transform' === $property && ! isset( $field['default']['text-transform'] ) ) ||
			( 'text-decoration' === $property && ! isset( $field['default']['text-decoration'] ) ) ||
			( 'color' === $property && ! isset( $field['default']['color'] ) ) ||
			( 'font-weight' === $property && ! isset( $field['default']['variant'] ) && ! isset( $field['default']['font-weight'] ) ) ||
			( 'font-style' === $property && ! isset( $field['default']['variant'] ) && ! isset( $field['default']['font-style'] ) )
			) ) {
			continue;
		}
		$script .= ( $choice_condition && 'font-family' === $args['choice'] ) ? $webfont_loader : '';

		if ( 'font-family' === $property || ( isset( $args['choice'] ) && 'font-family' === $args['choice'] ) ) {
			$css .= 'fontFamilyCSS=fontFamily;if(0<fontFamily.indexOf(\' \')&&-1===fontFamily.indexOf(\'"\')){fontFamilyCSS=\'"\'+fontFamily+\'"\';}';
			$var  = 'fontFamilyCSS';
		}
		$var  = ( ( empty( $args['prefix'] ) ) ? '' : '\'' . $args['prefix'] . '\'+' ) . $var . ( ( empty( $args['units'] ) ) ? '' : '+\'' . $args['units'] . '\'' ) . ( ( empty( $args['suffix'] ) ) ? '' : '+\'' . $args['suffix'] . '\'' );
		$css .= 'css+=(\'\'!==' . $var . ')?\'' . $args['element'] . '\'+\'{' . $property . ':\'+' . $var . '+\';}\':\'\';';
	}

	$script .= $css;
	if ( isset( $args['media_query'] ) ) {
		$script .= 'css=\'' . $args['media_query'] . '{\'+css+\'}\';';
	}
	return array(
		'script' => $script,
		'css'    => 'css',
	);
}
```
But here is a small mistype in webfont loader call: missed semicolon. Just check out screenshot: 
<img width="1356" alt="devtools - asra-wordpress wp-admin customize php return 2fwp-admin 2fplugins php 2018-09-22 20-54-50" src="https://user-images.githubusercontent.com/1572715/45920241-38b67700-beaa-11e8-8ee5-def70eed6c97.png">

### Version used:
3.0.34.1

**Comment by niklasp** [comment]
can be labeled "bug" possibly #2026

**Comment by aristath** [comment]
Fixed, will be included in 3.0.35
